composite_purchase_order.json

```diff
@@ -66,6 +70,34 @@
       "description": "indicates this purchase order should be re-encumbered each fiscal year. Only applies to ongoing orders",
       "type": "boolean"
     },
+    "renewal_cycle": {
+      "description": "length of the renewal cycle",
+      "type": "string",
+      "enum": [
+        "Two Years",
+        "One Year",
+        "3 Months",
+        "6 Months",
+        "1 Month",
+        "Manual"
+      ]
+    },
+    "renewal_interval": {
+      "description": "renewal interval (in days?)",
```

Comment:
Possible typo `?` I think renewal interval is in days so `?` is not required

Reply:
It was a question from the previous schema, Will have to confirm with Dennis